OpenRA.Mods.Common/Widgets/ProductionPaletteWidget.cs

```diff
@@ -484,7 +484,12 @@ public override void Draw()
 							icon.Pos + timeOffset,
 							Color.White, Color.Black, 1);
 
-					if (total > 1 || waiting)
+					if (first.Infinite)
+						/// Only regular has infinite symbol!
+						Game.Renderer.Fonts["Regular"].DrawTextWithContrast("∞",
+							icon.Pos + queuedOffset,
```

Comment:
The symbol jumps down noticably.  It would help polish if an extra offset could be applied here to center it vs the text. You can use SpriteFont.Measure().Y to find the relative heights of the different symbols.